### PR TITLE
fix(blog): Add notice, remove slug

### PR DESCRIPTION
**Pull request by boxbeam**

### CODE REVIEW

File: website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ---
-slug: running-tabby-locally-with-rocm.md
 title: Running Tabby Locally with AMD ROCm
 authors: [boxbeam]
 tags: [deployment]
 ---
 
+⚠️ ROCm is only supported in Tabby's nightly builds! Support will be stable in version 0.8.
```
Comment (wsxiaoys) [inline website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md:7]: add a info block https://docusaurus.io/docs/next/markdown-features/admonitions

Comment (wsxiaoys) [reply website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md:7]: (also make nightly text a link to nightly release page on github)

Comment (boxbeam) [reply website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md]: GitHub uses a different syntax for their extension to markdown, I've used that

Comment (wsxiaoys) [reply website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md]: but we're using docusaurus to render the blog post 👀

File: website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-slug: running-tabby-locally-with-rocm.md
 title: Running Tabby Locally with AMD ROCm
 authors: [boxbeam]
 tags: [deployment]
 ---
 
+:::warning
+
+Tabby's ROCm support is currently only in our [nightly builds](https://github.com/TabbyML/tabby/releases/tag/nightly). It will become stable in version 0.8.
+
```
Comment (wsxiaoys) [inline website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md:10]: there's also a closing annotation

image

Comment (boxbeam) [reply website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md:10]: Thank you, didn't notice that. I'll have to look for a way to preview this next time I'm working on a blog post.

**Pull request merged**